vuepress/docs/v7.0/docs/reference/deployment-structure.md

```diff
@@ -4,154 +4,128 @@ sidebarDepth: 2
 
 # Entando Deployment Structure
 
-This page provides a high level overview of the key Entando GitHub repositories along with a brief description 
-of how those repositories are realized in a running Entando deployment. The descriptions provided here are meant 
-as a guide for identifying opportunities to dig deeper into the architecture and how things are structured 
-rather than a panacea for understanding the architecture.
+This page provides an overview of the key Entando GitHub repositories with a brief description 
+of how these repositories are realized in a running Entando deployment. It also explores the architecture behind Entando and how its functions are structured.
 
 ## entando-operator
-The Entando operator coordinates the installation and configuration of all of the components of an Entando 
-Cluster. The operator can be installed once per Entando Cluster and used to coordinate the plugin lifecycle for 
-multiple Entando applications across many namespaces.
+The Entando Operator coordinates the installation and configuration of all of the components of an Entando 
+cluster. The operator can be installed once per cluster and used to coordinate the plugin lifecycle for multiple Entando applications across many namespaces.
 
 * GitHub: <https://github.com/entando-k8s/entando-k8s-controller-coordinator/>
 * DockerHub: <https://hub.docker.com/repository/docker/entando/entando-k8s-controller-coordinator>
 
 #### Customization
-It is unlikely that the operator will be customized as part of an Entando implementation. It is not built to 
-be extended inside the codebase. The most common pattern will be to use the existing custom resources that the 
-operator knows how to deploy to extend the Entando platform.
+Use existing [Entando Custom Resources](../consume/custom-resources.md) to extend the Entando platform.
```

Comment:
didn't we go with custom resources? like how "custom resource definitions" is lowercase, and so is "custom resources" when not directly preceded by Kubernetes?

Reply:
Entando Platform